launcher.py

```diff
@@ -65,10 +137,63 @@ def deploy_contract(self) -> bool:
         )
         return False
 
+    def clean_config_entry(self, line: str) -> str:
+        ''' Clean the line that forms the entry for the config.exs
+        '''
+        line = re.sub(r'"', '', line)
```

Comment:
`line.replace('"'. '')` is more conservative approach here

Reply:
Addressed in 41f2545